cmd/taskmasterd/fork.go

```diff
@@ -9,7 +9,14 @@ import (
 func fork() (int, error) {
 	logFile := logGetFile()
 
-	cmd := exec.Command(os.Args[0], "-d")
+	forkArgs := []string{
+		"-d",
+	}
+	for _, parent := range os.Args[1:] {
```

Comment:
If we were not piggies we would have instanciated a slice of `len(os.Args[1:]) + 1` length 👀 .